homeassistant/components/roborock/switch.py

```diff
@@ -84,6 +84,24 @@ class RoborockSwitchDescription(
         icon="mdi:bell-cancel",
         entity_category=EntityCategory.CONFIG,
     ),
+    RoborockSwitchDescription(
```

Comment:
Does this come with a "default" time like DnD or what happens if no time exist?

Reply:
There is a "default" time specified in the app. But I'm not sure what it is. There should always be some value set, it shouldn't ever be None